src/main/java/io/github/moulberry/notenoughupdates/overlays/EquipmentOverlay.java

```diff
@@ -303,8 +319,18 @@ private void updateGuiInfo(GuiScreen screen) {
 
 	private void drawSlot(ItemStack stack, int x, int y, int mouseX, int mouseY, List<String> tooltip) {
 		if (stack == null) return;
+
 		Utils.drawItemStack(stack, x, y, true);
 		if (Utils.isWithinRect(mouseX, mouseY, x, y, 16, 16)) {
+			// draw the slot overlay
+			GlStateManager.disableLighting();
+			GlStateManager.disableDepth();
+			GlStateManager.colorMask(true, true, true, false);
+			Utils.drawGradientRect(x, y, x + 16, y + 16, -2130706433, -2130706433);
+			GlStateManager.colorMask(true, true, true, true);
+			GlStateManager.enableLighting();
+			GlStateManager.enableDepth();
```

Comment:
Maybe move this entire highlighting code into a subroutine, since you call it from multiple places and it is quite long

Reply:
moved to `Utils#drawHoverOverlay`